data/ sql/ updates/ pending_db_world/rev_1559170250353381400.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1559170250353381400');
+
+UPDATE `creature_template` SET `flags_extra` = 128 WHERE (entry = 18374);
```

Comment:
```suggestion
UPDATE `creature_template` SET `flags_extra` = `flags_extra` | 128 WHERE (entry = 18374);
```

I should have suggested this to other people before too but it's the best way for flags. In case someone have modified the flag, it will not erase it but it will make sure it has 128. Example, if it has 2, then it will have 130